packages/graphql/tests/schema/directives/relationship-nested-operations.test.ts

```diff
@@ -5233,6 +5235,7 @@ describe("Relationship nested operations", () => {
                   movies(options: MovieOptions, where: MovieWhere): [Movie!]!
                   moviesAggregate(where: MovieWhere): MovieAggregateSelection!
                   moviesConnection(after: String, first: Int, sort: [MovieSort], where: MovieWhere): MoviesConnection!
+                  people(options: QueryOptions, where: PersonWhere): [Person!]!
```

Comment:
Do we have introduced top-level union queries? can you double-check this?

Reply:
yes, we have simple queries for unions implemented. There is no sorting supported on them.